lib/ansible/modules/system/xfconf.py

```diff
@@ -152,61 +158,78 @@ def main():
     # Assign module values to dictionary values
     channel = module.params['channel']
     property = module.params['property']
-    value_type = module.params['value_type']
-    if module.params['value'].lower() == "true":
-        value = "true"
-    elif module.params['value'] == "false":
-        value = "false"
+    values = module.params['value']
+    value_types = module.params['value_type']
```

Comment:
After this line, add:
```
if type(values) in ('bool', 'int', 'float', 'str'):  # I may be missing some types here
    values = [values]
if type(value_types) in ('bool', 'int', 'float', 'str'):
    values = [value_types]
```
With that we guarantee that whatever goes into the rest of the code is a list